go.mod

```diff
@@ -65,6 +65,8 @@ require (
 	github.com/lib/pq v1.10.9
 	github.com/linkedin/goavro/v2 v2.12.0
 	github.com/manifoldco/promptui v0.9.0
+	github.com/marcboeker/go-duckdb v1.5.1
+	github.com/minio/minio-go v6.0.14+incompatible
```

Comment:
who is using this?

Reply:
> 	github.com/minio/minio-go v6.0.14+incompatible
I was still using `minio-go` instead of `minio-go/v7`. Fixed it.

Reply:
> github.com/marcboeker/go-duckdb v1.5.1

This is being used in [here](https://github.com/rudderlabs/rudder-server/blob/86257efec90ec3b4290bbbfa7f8d8db9a4b0148b/enterprise/reporting/error_index/worker_test.go#L493).

Reply:
you need to fix compilation errors now!

Reply:
Soddy, Didn't realize I am still using `destination.MINIOResource` in few more places. Fixing now.